### ISSUE TITLE
Jazz up the home page 

### ISSUE THREAD

**Issue by dalanmiller** [giladgray/WhichBus#33]
I think using this .js lib (http://www.sequencejs.com/themes/apple-style/) we can make the home page a bit more snazzy. 

Maybe the bus can roll in and then go ka-chunk ka-chunk as it's idling lol?

**Comment by paigepauli** [comment]
I agree, it looks cool, but aren't we looking to simply have a clean UI
instead of advertising our mobile product? It looks more like it's for
marketing a product instead of being implemented into a site that
_does_ something
like ours. Thoughts?

On Thu, Feb 16, 2012 at 1:23 PM, Daniel Miller <
reply@reply.github.com
